redpanda/tests/acceptance_test.go

```diff
@@ -22,7 +23,7 @@ const (
 	gcpDedicatedClusterFile   = "../../examples/cluster/gcp/main.tf"
 	serverlessClusterFile     = "../../examples/cluster/serverless/main.tf"
 	awsByocClusterFile        = "../../examples/byoc/aws/main.tf"
-	awsByocVpcClusterFile     = "../../examples/byovpc/aws/main.tf"
+	awsByocVpcClusterFile     = "../../infra/byovpc/aws/main.tf"
```

Comment:
So, overall this seems fine to me, but TBH I'm not thrilled about the top-level `infra` directory. Is there a reason not to put this somewhere like `redpanda/tests/infra`?